openwisp_controller/config/base/config.py

```diff
@@ -318,6 +318,25 @@ def enforce_required_templates(cls, action, instance, pk_set, **kwargs):
                     *required_templates.order_by('name').values_list('pk', flat=True)
                 )
 
+    @classmethod
+    def certificate_renewed(cls, instance, created, **kwargs):
+        if created:
+            return
+        try:
+            config = instance.vpnclient.config
+        except ObjectDoesNotExist:
+            return
+        else:
+            transaction.on_commit(
+                lambda: config_modified.send(
```

Comment:
I had a doubt, does this also set the config status as modified?
If not, we could simply call `config.set_status_modified()`.

Reply:
Yes it does, but I changed it to use `config.set_status_modified()` to keep code simple.